start

```diff
@@ -233,6 +233,7 @@ function copy_defaults() {
       cp /opt/stellar-default/$NETWORK/supervisor/etc/supervisord.conf.d/soroban-rpc.conf $SUPHOME/etc/supervisord.conf.d 2>/dev/null || true
     fi
     cp /opt/stellar-default/$NETWORK/supervisor/etc/supervisord.conf.d/friendbot.conf $SUPHOME/etc/supervisord.conf.d 2>/dev/null || true
+    cp /opt/stellar-default/$NETWORK/supervisor/etc/supervisord.conf.d/history-archive.conf $SUPHOME/etc/supervisord.conf.d 2>/dev/null || true
```

Comment:
Consider replacing cp with install, as it's working around files in use.

Reply:
What do you mean by "install"? I'm not familiar with that.

Reply:
Going to merge because I'm not sure how to change to doing an install, and the line I'm adding here is consistent with how existing services confs get added. Let me know how to do an install and happy to follow up in another PR.

Reply:
the linux `install` command is similar to `cp`, but act as 
1. `rm dest`
2. then, `cp src dest`

The main advantage in using it is to avoid the potential failure of the `cp` command in case the file is being used.
( there are other benefits, but this is the important one, imho )

Reply:
TIL. It also sounds like the other main reason to use install is that it removes the hard link meaning the copy will only replace the file at that one path, not files at all paths sharing the hardlink.

It's very unlikely the file is in use, and equally unlikely the file would have multiple hard links, so I don't think using install will have a material difference.

A downside of install is similar to sed, it's behavior is different on GNU vs BSD linux, so fun times if we encourage use and it ends up in scripts run on Linux and Mac.